### DISCUSSION TITLE
How can I get the user information in response after authentication

### DISCUSSION THREAD

**Discussion by NaimaGhulamMuhammad** [Web3Auth/Web3Auth#793] [Help]
can I get the user detail like username, email in response to authentication,
I have read documentation, no information is provided for the response, and in dashboard its just the number of users only.

Please be patient, I am an student and trying to learn

Note: I am using web3auth via moralis, but trying to get user twitter details.

**Comment by shahbaz17** [comment]
Hey @NaimaGhulamMuhammad 

You can get the userInfo using the below function after a user is logged in:
```js
await web3auth.getUserInfo();
```

Please check out https://web3auth.io/docs/sdk/web/modal/usage#get-users-information to learn more.
